ttnn/ttnn/distributed/distributed.py

```
@@ -38,6 +38,12 @@ def _get_rich_table(
         logger.error("Error getting device mesh shape: {}.", e)
         rows, cols = 0, 0
 
+    try:
```

I think we can just throw here - remove `try` / `except` to avoid swallowing the exception. `view` should always be present on mesh device